packages/@react-spectrum/button/stories/Button.stories.tsx

```diff
@@ -105,36 +73,44 @@ storiesOf('Button', module)
     () => renderStyles()
   );
 
-function render<T extends ElementType = 'button'>(props: SpectrumButtonProps<T> = {variant: 'primary'}) {
-  return (
+function render<T extends ElementType = 'button'>(props: SpectrumButtonProps<T> & {showIconWithText?: boolean} = {variant: 'primary'}) {
+  let {
+    showIconWithText,
```

Comment:
Does this need to be a control? Should we just render one with and one without always?